matlab/lib/+Ice/AlreadyRegisteredException.m

```diff
@@ -9,9 +9,8 @@
 %   id - The ID (or name) of the object that is registered already.
 
 % Copyright (c) ZeroC, Inc. All rights reserved.
-% Generated from LocalException.ice by slice2matlab version 3.7.10
 
-classdef AlreadyRegisteredException < Ice.LocalException
+classdef (Sealed) AlreadyRegisteredException < Ice.LocalException
```

Comment:
This is one of the few exceptions we throw from MATLAB code (there is also MarshalException, UnknownUserException, TwowayOnlyException).

These "special" exceptions have a more convenient constructor where you don't need to specify the errID. 

For this constructor, you don't need to specify "what" either as it's always computed.